components/crud-web-apps/jupyter/backend/apps/common/form.py

```diff
@@ -333,3 +336,9 @@ def add_notebook_volume(notebook, vol_name, claim, mnt_path):
     # Container Mounts
     mnt = {"mountPath": mnt_path, "name": vol_name}
     container["volumeMounts"].append(mnt)
+
+def set_notebook_language(notebook, body, defaults):
+    lang = get_form_value(body, defaults, "language")
+    if not re.match('^[a-zA-Z]{2}(_[a-zA-Z]{2})?$', lang):
+        raise BadRequest("Error: the value of KF_LANG environment variable ('" + lang + "') is not a valid format (e.g 'en', 'en_US', ...)")
```

Comment:
This message needs to be i18n. Look at how other messages are i18n within the code for help.